scripts/tools/initialize_virtualenv.py

```diff
@@ -112,7 +112,7 @@ def main():
     Setup local virtual environment.
     """
     airflow_home_dir = Path(os.environ.get("AIRFLOW_HOME", Path.home() / "airflow"))
-    airflow_sources = Path(__file__).parents[2]
+    airflow_sources = Path(__file__).parents[2].absolute()
```

Comment:
Do not use `absolute`, it is undocumented and known to be erroneous in edge cases.

Reply:
Thanks! I've replaced it with `resolve`